.github/workflows/ds.yml

```diff
@@ -1,36 +0,0 @@
-name: DS Tests
-
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-  workflow_dispatch:
-
-concurrency: 
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: true
-
-env:
-  CARGO_TERM_COLOR: always
-
-jobs:
-  tests:
-    strategy:
-      fail-fast: false
-      matrix:
-        os:
-          - macos-latest
-          - ubuntu-latest
-          - windows-latest
-    runs-on: ${{ matrix.os }}
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          ref: ${{ github.event.pull_request.head.sha }}
-      - uses: dtolnay/rust-toolchain@stable
-      - uses: Swatinem/rust-cache@v2
-      - name: Test DS lib
-        run: cargo test --verbose -p ds-lib
```

Comment:
We don't run these tests anymore. Not sure if that's a big deal for DS though.

Reply:
Good point. As discussed off-GitHub I'll add a `cargo test` afterwards.